src/transforms/onehot.jl

```diff
@@ -69,7 +78,6 @@ function revertfeat(::OneHot, newfeat, fcache)
   end
 
   ocolumn = categorical(x; levels, ordered)
-
```

Comment:
You are changing the whitespace of the code, deleting break lines. Please avoid that.

Reply:
Okay 👍I will maintain the break lines.